modules/editorial-comments/editorial-comments.php

```diff
@@ -6,117 +6,118 @@
+			add_action( 'admin_enqueue_scripts', array( $this, 'add_admin_scripts' ) );
+			add_action( 'wp_ajax_editflow_ajax_insert_comment', array( $this, 'ajax_insert_comment' ) );
 		}
 
-	}
+		/**
+		 * Upgrade our data in case we need to
+		 *
+		 * @since 0.7
+		 */
+		public function upgrade( $previous_version ) {
+			global $edit_flow;
+
+			// Upgrade path to v0.7
+			if ( version_compare( $previous_version, '0.7', '<' ) ) {
+				// Technically we've run this code before so we don't want to auto-install new data
+				$edit_flow->update_module_option( $this->module->name, 'loaded_once', true );
+			}
+		}
 
-	/**
-	 * Load any of the admin scripts we need but only on the pages we need them
-	 */
-	function add_admin_scripts( ) {
-		global $pagenow;
+		/**
+		 * Load any of the admin scripts we need but only on the pages we need them
+		 */
+		public function add_admin_scripts() {
+			global $pagenow;
 
-		$post_type = $this->get_current_post_type();
-		$supported_post_types = $this->get_post_types_for_module( $this->module );
-		if ( !in_array( $post_type, $supported_post_types ) )
-			return;
+			$post_type = $this->get_current_post_type();
+			$supported_post_types = $this->get_post_types_for_module( $this->module );
+			if ( ! in_array( $post_type, $supported_post_types ) ) {
+				return;
+			}
 
-		if ( !in_array( $pagenow, array( 'post.php', 'page.php', 'post-new.php', 'page-new.php' ) ) )
-			return;
+			if ( ! in_array( $pagenow, array( 'post.php', 'page.php', 'post-new.php', 'page-new.php' ) ) ) {
+				return;
+			}
 
-		wp_enqueue_script( 'edit_flow-post_comment', $this->module_url . 'lib/editorial-comments.js', array( 'jquery', 'wp-ajax-response' ), EDIT_FLOW_VERSION, true );
-		wp_localize_script( 'edit_flow-post_comment', '__ef_localize_post_comment', array(
-			'and'           => esc_html__( 'and', 'edit-flow' ),
-			'none_notified' => esc_html__( 'No one will be notified.', 'edit-flow' ),
-		) );
+			wp_enqueue_script( 'edit_flow-post_comment', $this->module_url . 'lib/editorial-comments.js', array( 'jquery', 'wp-ajax-response' ), EDIT_FLOW_VERSION, true );
+			wp_localize_script( 'edit_flow-post_comment', '__ef_localize_post_comment', array(
+				'and'           => esc_html__( 'and', 'edit-flow' ),
+				'none_notified' => esc_html__( 'No one will be notified.', 'edit-flow' ),
+			) );
 
-		wp_enqueue_style( 'edit-flow-editorial-comments-css', $this->module_url . 'lib/editorial-comments.css', false, EDIT_FLOW_VERSION, 'all' );
+			wp_enqueue_style( 'edit-flow-editorial-comments-css', $this->module_url . 'lib/editorial-comments.css', false, EDIT_FLOW_VERSION, 'all' );
 
-		$thread_comments = (int) get_option('thread_comments');
-		?>
+			$thread_comments = (int) get_option( 'thread_comments' );
+			?>
 		<script type="text/javascript">
-			var ef_thread_comments = <?php echo ($thread_comments) ? $thread_comments : 0; ?>;
+			var ef_thread_comments = <?php echo esc_html( ( $thread_comments ) ? $thread_comments : 0 ); ?>;
```

Comment:
Minor, could use `intval` for sanitization here to be more strict and specific:

```suggestion
			var ef_thread_comments = <?php echo $thread_comments ? intval( $thread_comments ) : 0; ?>;
```

Reply:
Done